_ml-commons-plugin/custom-local-models.md

```diff
@@ -321,148 +321,3 @@ The response contains the tokens and weights:
 ## Step 5: Use the model for search
 
 To learn how to use the model for vector search, see [Using an ML model for neural search]({{site.url}}{{site.baseurl}}/search-plugins/neural-search/#using-an-ml-model-for-neural-search).
-
-## Cross-encoder models
-
-Cross-encoder models support query reranking. 
-
-To register a cross-encoder model, send a request in the following format. The `model_config` object is optional. For cross-encoder models, specify the `function_name` as `TEXT_SIMILARITY`. For example, the following request registers an `ms-marco-TinyBERT-L-2-v2` model:
-
-```json
-POST /_plugins/_ml/models/_register
-{
-    "name": "ms-marco-TinyBERT-L-2-v2",
-    "version": "1.0.0",
-    "function_name": "TEXT_SIMILARITY",
-    "description": "test model",
-    "model_format": "TORCH_SCRIPT",
-    "model_group_id": "lN4AP40BKolAMNtR4KJ5",
-    "model_content_hash_value": "90e39a926101d1a4e542aade0794319404689b12acfd5d7e65c03d91c668b5cf",
-    "model_config": { 
-        "model_type": "bert",
-        "embedding_dimension": 1,
-        "framework_type": "huggingface_transformers",
-        "total_chunks":2,
-        "all_config": "{\"total_chunks\":2}"
-    },
-    "url": "https://github.com/opensearch-project/ml-commons/blob/main/ml-algorithms/src/test/resources/org/opensearch/ml/engine/algorithms/text_similarity/TinyBERT-CE-torch_script.zip?raw=true"
-}
```

Comment:
Is the `TEXT_SIMILARITY` function name documented anywhere else? If not, we should find a place

Reply:
Makes sense. I added the types to the register model API and to the custom local models page.